rfcs/InputUnion.md

```diff
@@ -288,11 +284,27 @@ type Mutation {
 }
 ```
 
-##### Variations:
+#### Variations:
 
 * A `default` type may be defined, for which specifying the `__typename` is not required. This enables a field to migration from an `Input` to an `Input Union`
 
-#### Single user-chosen field; value is the `type`
+#### Evaluation
+
+- A:
```

Comment:
Perhaps the letter could be a link to the solution...

Reply:
I decided to include the title and link it. Switching back and forth makes it really hard to navigate.